casper/src/main/scala/coop/rchain/casper/genesis/Genesis.scala

```diff
@@ -35,7 +36,8 @@ object Genesis {
   def defaultBlessedTerms(
       posParams: ProofOfStake,
       vaults: Seq[Vault],
-      shardId: String
+      shardId: String,
+      configRegistry: ConfigRegistry
```

Comment:
Better place this next to `ProofOfStake` as they are both compiled Rholang templates